tests/test_feature_extraction_common.py

```diff
@@ -266,7 +266,7 @@ def test_padding_accepts_tensors_tf(self):
         input_np = feat_extract.pad(processed_features, padding="longest", return_tensors="np")[input_name]
         input_tf = feat_extract.pad(processed_features, padding="longest", return_tensors="tf")[input_name]
 
-        self.assertTrue(abs(input_np.sum() - input_tf.numpy().sum()) < 1e-2)
+        self.assertTrue(abs(input_np.astype(np.float32).sum() - input_tf.numpy().sum()) < 1e-2)
```

Comment:
This test was failing sometimes for S2T because here `input_np` is of type `float64` but `input_tf.numpy()/input_pt.numpy().sum())` is `float32`, so casting `input_np` to `float32`